.travis.yml

```diff
@@ -33,7 +33,7 @@ git:
 
 install:
   - >
-    export IRIS_TEST_DATA_REF="dba47566a9147645fea586f94a138e0a8d45a48e";
+    export IRIS_TEST_DATA_REF="5d4eab1b4b3f6ef9ede5c0d24853c902f3613fea";
```

Comment:
N.B. ok, but this will need changing *again* when we merge https://github.com/SciTools/iris-test-data/pull/53,
because this commit is on your branch.

Stop press: 
https://github.com/SciTools/iris-test-data/pull/53 **now merged !**
New master commit SHA  is `1696ac3a823a06b95f430670f285ee97671d2cf2`
So that's what is now needed here ...